core/themes/basis/css/component/comment.css

```diff
@@ -163,3 +163,16 @@
 [dir="rtl"] .comment-footer {
   float: left;
 }
+
+a.comment-permalink  {
+  width: .7em;
+  height: .7em;
+  padding: .1em;
+  display: inline-block;
+  background-color: currentColor;
+  -webkit-mask-image: url("/core/modules/comment/images/link-icon.svg");
+  mask-image: url("/core/modules/comment/images/link-icon.svg");
```

Comment:
Unless I'm missing something, the png variant isn't used anymore and could get deleted from the images directory.

Reply:
@indigoxela you are 100% correct. Can't believe I didn't catch this. I think you pointed this out in the past. Will fix.